pkgs/development/python-modules/afdko/default.nix

```
@@ -14,13 +33,13 @@
 
 buildPythonPackage rec {
   pname = "afdko";
-  version = "3.9.0";
+  version = "3.9.2";
```

```suggestion
  version = "3.9.2";
  format = "pyproject";
```

Move the format up here.